### PR TITLE
Update website server code according to style guides

### PR DESCRIPTION
**Pull request by enaqx**
If it is OK, there is a lot of style fixing in other places.

### CODE REVIEW

**Comment by brentvatne** [comment]
@enaqx - were you able to build the site without any errors after these changes?

**Comment by brentvatne** [comment]
@enaqx - I haven't heard back from you for a few weeks so I'm going to close this, ping me if you're interested in finishing this up and I'll reopen!

**Comment by enaqx** [comment]
@brentvatne ping-ping. I've checked again these changes and site is working on my side. Can you check, pls? If you would like any proves (logs, screenshots, etc.) I would be happy to make it. I'm open to any suggestions or corrections. :smile:

File: website/server/convert.js [hunk 1]
```diff
@@ -7,6 +7,8 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  */
```
Comment (ide) [inline website/server/convert.js]: remove newline

File: website/server/convert.js [hunk 2]
```diff
@@ -76,7 +80,8 @@ function execute() {
     }
 
     // Create a dummy .js version that just calls the associated layout
-    var layout = metadata.layout[0].toUpperCase() + metadata.layout.substr(1) + 'Layout';
+    var layout = metadata.layout[0].toUpperCase()
```
Comment (ide) [inline website/server/convert.js]: + operator goes at the end of the line above

**Comment by enaqx** [comment]
@ide Thanks for style corrections. Since I've removed this branch and GitHub will not allow changes in this commit I'll open new PR with updates. Is it OK, guys?

**Comment by ide** [comment]
Sounds good.

**Pull request closed without merging**